keyboards/ungodly/trinity/info.json

```diff
@@ -0,0 +1,861 @@
+{
+    "manufacturer": "Ungodly Design",
+    "keyboard_name": "Trinity",
+    "maintainer": "Luis Godinez",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "usb": {
+        "vid": "0x3530",
+        "pid": "0x6644",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["GP29", "GP20", "GP19", "GP12", "GP9", "GP1"],
+        "cols": ["GP28", "GP27", "GP26", "GP25", "GP24", "GP23", "GP21", "GP2", "GP13", "GP10", "GP11", "GP8", "GP14", "GP6", "GP5", "GP4", "GP3", "GP0"]
+    },
+    "community_layouts": ["tkl_ansi", "tkl_ansi_split_bs_rshift", "tkl_ansi_tsangan", "tkl_ansi_tsangan_split_bs_rshift", "tkl_iso", "tkl_iso_split_bs_rshift", "tkl_iso_tsangan", "tkl_iso_tsangan_split_bs_rshift"]
```

Comment:
```suggestion
    "community_layouts": ["tkl_ansi", "tkl_ansi_split_bs_rshift", "tkl_ansi_tsangan", "tkl_ansi_tsangan_split_bs_rshift", "tkl_ansi_wkl", "tkl_ansi_wkl_split_bs_rshift", "tkl_iso", "tkl_iso_split_bs_rshift", "tkl_iso_tsangan", "tkl_iso_tsangan_split_bs_rshift", "tkl_iso_wkl", "tkl_iso_wkl_split_bs_rshift"],
```

Reply:
Fixed in https://github.com/qmk/qmk_firmware/pull/23577/commits/38116b90863cd82398ad59245152ae9ab581001c